tests/console/consoletest_finish.pm

```diff
@@ -74,7 +74,14 @@ sub run() {
         }
         wait_idle;
         mouse_hide(1);
-        assert_screen 'generic-desktop';
+        # in case of awesome test we use xdm and need to log in at the display
+        # manager
+        if (check_var("DESKTOP", "awesome")) {
+            assert_screen 'displaymanager', 200;
+        }
```

Comment:
see my comments in #930 - asserting here for displaymanager is wrong. At this point we *need* to be at the desktop